src/audio-worklet/free-queue/README.md

```diff
@@ -0,0 +1,11 @@
+# Free Queue 
+
+A Lock-Free Ring Buffer Implementation, focused on utility for applications related to Audio Processing with WebAudio API.
+
+## Working
+
+It is a Lock-Free implementation based on Single Producer - Single Consumer Concept.
+
+This Library Provieds API for creating a FreeQueue instance, to which data can be pushed to or pulled from by different worker threads.
```

Comment:
Use back tick single quotes (`) for the code display in the text:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code